includes/admin/class-wc-stripe-upe-compatibility-controller.php

```diff
@@ -63,7 +63,7 @@ public function add_compatibility_notice() {
 				],
 				[
 					'name'         => 'WooCommerce',
-					'version'      => $this->get_wc_version(),
+					'version'      => $this->get_wc_version() ? $this->get_wc_version() : '[version-not-found]',
```

Comment:
Assuming `WC_VERSION` always exists, this can probably be just `$this->get_wc_version()` as well.

Reply:
You're right, I missed that 💯 
```suggestion
					'version'      => $this->get_wc_version(),
```